src/core/sys/dragonflybsd/sys/elf32.d

```diff
@@ -0,0 +1,187 @@
+/**
+ * D header file for DragonFlyBSD.
+ *
+ * Authors: Diederik de Groot(port:DragonFlyBSD)
+ * Copied:  From core/sys/freebsd/sys
+ */
+module core.sys.dragonflybsd.sys.elf32;
+
+version (DragonFlyBSD):
+
+extern (C) pure nothrow @nogc @system:
```

Comment:
No functions other than the few you define below with different attributes, so all attributes other than `extern (C)` can be removed.